<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="vSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="leText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="leTextCopy">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
